scripts/syntactics/index.rst

```diff
@@ -0,0 +1,78 @@
+Deep Biaffine Dependency Parser
+---------------------------------
+
+This package is an implementation of `Deep Biaffine Attention for Neural Dependency Parsing <https://arxiv.org/pdf/1611.01734.pdf>`_ proposed by Dozat and Manning (2016), with SOTA accuracy.
```

Comment:
The download link needs to be added back. The zipfile name is typically the same as the folder name (i.e. `syntactics.zip`). Was there any issue when you added it?

Reply:
The download file is automatically generated during the build, so you just need to put syntactics.zip in the download. It doesn't need to be uploaded elsewhere.